doc/ale-haml.txt

```diff
@@ -0,0 +1,32 @@
+===============================================================================
+ALE HAML Integration                                         *ale-haml-options*
+
+
+===============================================================================
+hamllint                                                    *ale-haml-hamllint*
+
+Website: https://github.com/brigade/haml-lint
+
+
+Installation
+-------------------------------------------------------------------------------
+
+Install haml-lint globally:
+
+  gem install haml_lint
+
+
+Options
+-------------------------------------------------------------------------------
+
+g:ale_haml_hamllint_options                       *g:ale_haml_hamllint_options*
+
+  Type: |String|
+  Default: `''`
+
+  This variable can be set and will be passed to haml-lint as options
+  eg. let g:ale_haml_hamllint_options = '-c ~/use-my.haml-lint.yml'
```

Comment:
This could use some a bit of formatting for the example. I recommend ending your sentence on "options" with a full stop (or "period" in American English), and then write "For example:" and a little code block. Have a look at `:help g:ale_enabled`. That's one option which has some formatting like that.